data/items.ts

```diff
@@ -6636,7 +6651,39 @@ export const Items: {[itemid: string]: ItemData} = {
 		fling: {
 			basePower: 60,
 		},
-		// Implemented in statuses.js, moves.js, and abilities.js
+		// Partially implemented in Pokemon.effectiveWeather() in sim/pokemon.ts
+		onStart(pokemon) {
+			pokemon.addVolatile('utilityumbrella');
```

Comment:
Not sure what the point of this volatile is when it seems that you could add the relevant event handlers directly to the item.